common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -143,6 +143,10 @@ public void setValue(Object value) {
         userEdited = true;
     }
 
+    void restoreValue(Object value) {
+        setValue((T) value);
+    }
```

Comment:
Yes, and no. It is needed in this patch, since I could not find any other way to cast the argument to T when reading the value back from a file.

But in my working branch I have unified this behavior with Storage under the same name, as well.